docs/custom-flows/overview.mdx

```diff
@@ -22,32 +28,34 @@ The following steps outline the sign-up process:
 
 | Name | Description |
 | --- | --- |
 | `requiredFields` | All fields that must be collected before the `SignUp` converts into a `User`. |
 | `optionalFields` | All fields that can be collected, but are not necessary to convert the `SignUp` into a `User`. |
-| `missing_fields` | A subset of `required_fields`. It contains all fields that still need to be collected before a `SignUp` can be converted into a `User`. Note that this property will be updated dynamically. As you add more fields to the `SignUp`, they will be removed. Once this property is empty, your `SignUp` will automatically convert into a `User`. |
+| `missingFields` | A subset of `requiredFields`. It contains all fields that still need to be collected before a `SignUp` can be converted into a `User`. Note that this property will be updated dynamically. As you add more fields to the `SignUp`, they will be removed. Once this property is empty, your `SignUp` will automatically convert into a `User`. |
 
-### Verified fields
+#### Verified fields
 
-Some fields of the `SignUp`, such as `email_address` and `phone_number`, must be verified before they are fully added to the `SignUp`. Similar to what happens with required fields, the `SignUp` contains the current state of all verified fields.
+Some properties of the `SignUp`, such as `emailAddress` and `phoneNumber`, must be **verified** before they are **fully** added to the `SignUp` object.
 
-The following keys relate to the verification state of the `SignUp`:
+The `SignUp` object will show **the state of verification** in the following properties:
 
 | Name | Description |
 | --- | --- |
 | `unverifiedFields` | A list of all [`User`](/docs/references/javascript/user/user) attributes that need to be verified and are pending verification. This is a list that gets updated dynamically. When verification for all required fields has been successfully completed, this value will become an empty array. |
 | `verifications` | An object that describes the current state of verification for the [`SignUp`](/docs/references/javascript/sign-in/sign-in). There are currently three different keys: `email_address`, `phone_number`, and `external_account`. |
 
-## Sign-in flow
+### Sign-in flow
 
-Sign-in's are initiated by creating a [`SignIn`](/docs/references/javascript/sign-in/sign-in) object on the current `Client`. The `SignIn` handles all the state and logic associated with a sign-in. If the sign-in is successfully authenticated, it will transform into an active session on the current `Client`.
+The [`SignIn`](/docs/references/javascript/sign-in/sign-in) object is the pivotal concept in the sign-in process.
+
+Sign-in's are initiated by creating a `SignIn` object on the current `Client`. If the sign-in is successfully authenticated, it will transform into an active session for that [`User`](/docs/references/javascript/user/user) on the current `Client`.
```

Comment:
```suggestion
Sign-ins are initiated by creating a `SignIn` object on the current `Client`. If the sign-in is successfully authenticated, it will transform into an active session for that [`User`](/docs/references/javascript/user/user) on the current `Client`.
```

I don't thiiiink the apostrophe is correct here